[2025-03-01T19:28:24.336+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-01T19:28:24.395+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: redshift_query_dag.upload_to_s3 manual__2025-03-01T19:28:18.505895+00:00 [queued]>
[2025-03-01T19:28:24.424+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: redshift_query_dag.upload_to_s3 manual__2025-03-01T19:28:18.505895+00:00 [queued]>
[2025-03-01T19:28:24.425+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-01T19:28:24.456+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2025-03-01 19:28:18.505895+00:00
[2025-03-01T19:28:24.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9295) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-01T19:28:24.482+0000] {standard_task_runner.py:72} INFO - Started process 9302 to run task
[2025-03-01T19:28:24.480+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'redshift_query_dag', 'upload_to_s3', 'manual__2025-03-01T19:28:18.505895+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/final_aggregation_dag.py', '--cfg-path', '/tmp/tmp8fyep8zi']
[2025-03-01T19:28:24.484+0000] {standard_task_runner.py:105} INFO - Job 126: Subtask upload_to_s3
[2025-03-01T19:28:24.505+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-01T19:28:24.550+0000] {task_command.py:467} INFO - Running <TaskInstance: redshift_query_dag.upload_to_s3 manual__2025-03-01T19:28:18.505895+00:00 [running]> on host 763fac820cd7
[2025-03-01T19:28:24.670+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='redshift_query_dag' AIRFLOW_CTX_TASK_ID='upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T19:28:18.505895+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T19:28:18.505895+00:00'
[2025-03-01T19:28:24.672+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-01T19:28:24.704+0000] {credentials.py:1278} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2025-03-01T19:28:24.792+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-01T19:28:24.802+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-01T19:28:24.802+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=redshift_query_dag, task_id=upload_to_s3, run_id=manual__2025-03-01T19:28:18.505895+00:00, execution_date=20250301T192818, start_date=20250301T192824, end_date=20250301T192824
[2025-03-01T19:28:24.873+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-01T19:28:24.891+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T19:28:24.893+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
